.github/workflows/eamxx-kokkos-develop-testing.yml

```diff
@@ -0,0 +1,159 @@
+name: eamxx-sa-kokkos-develop
+
+on:
+  # Manual run is used to bless
+  workflow_dispatch:
+    inputs:
+      job_to_run:
+        description: 'Job to run'
+        required: true
+        type: choice
+        options:
+          - gcc-openmp
+          - gcc-cuda
+          - all
+      bless:
+        description: 'Generate baselines'
+        required: true
+        type: boolean
+      submit:
+        description: 'Force cdash submission'
+        required: true
+        type: boolean
+
+  # Add schedule trigger for nightly runs at midnight MT (Standard Time)
+  schedule:
+    - cron: '0 7 * * 1'  # Runs at 7 AM UTC on Mondays, which is midnight MT during Standard Time
```

Comment:
Does this look correct for running first thing Monday morning? WE could also do this nightly if we wanted, but probably overkill.